extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilderSuite.scala

```diff
@@ -434,7 +434,17 @@ abstract class PrivilegesBuilderSuite extends AnyFunSuite
     val accessType0 = ranger.AccessType(po0, operationType, isInput = true)
     assert(accessType0 === AccessType.SELECT)
 
-    assert(out.size === 0)
+    assert(out.size === 1)
+    val po1 = out.head
+    assert(po1.actionType === PrivilegeObjectActionType.OTHER)
+    assert(po1.privilegeObjectType === PrivilegeObjectType.TABLE_OR_VIEW)
+    assertEqualsIgnoreCase(reusedDb)(po1.dbname)
+    assertEqualsIgnoreCase(reusedPartTableShort)(po1.objectName)
+    // ignore this check as it behaves differently across spark versions
+    assert(po1.columns.isEmpty)
+    checkTableOwner(po1)
+    val accessType1 = ranger.AccessType(po1, operationType, isInput = true)
+    assert(accessType1 === AccessType.SELECT)
```

Comment:
Should be ALTER?

Reply:
> Should be ALTER?

Hmm. the operation type here is ANALYZE_TABLE, so the access type is select.
One query only one operation type

Reply:
+1 for ALTER.
Although the stat is just stats with no guarantee or accuracy or must-refreshing crossing different engines, they are still part of the table and should not be changed without any proper privilege checking. As we have ALTERTABLE_SERDEPROPERTIES in OperationType, it's a proper operation type for such metadata privilege checks.